CHANGELOG.md

```diff
@@ -1,6 +1,277 @@
 Changelog
 =========
 
+0.5.0
+-----
+
+**Date:** 2020-03-XX
+
+Perfectly Matched Layer (PML) and Bug Fixes
+
+This release introduces PML boundary conditions for the Yee field solver and particle 
+rendering with the ISAAC plugin.
+
+Thanks to Igor Andriyash, Sergei Bastrakov, Xeinia Bastrakova, Andrei Berceanu,
+Finn-Ole Carstens, Alexander Debus, Jian Fuh Ong, Marco Garten, Axel Huebl,
+Sophie Koßagk, Anton Lebedev, Felix Meyer, Pawel Ordyna, Richard Pausch,
```

Comment:
Sophie married two years ago and is now Sophie Rudat :)
```suggestion
Sophie Rudat, Anton Lebedev, Felix Meyer, Pawel Ordyna, Richard Pausch,
```

Reply:
Yea, already commented that but resolved it when @psychocoderHPC said that you can find her under that name on GitHub.
Ideally, you'd also want to find someone's work by their previous name and their current work and contact by their new name.
So `<first name> <new name> (<old name>)`  ... but IDK, that's how I would do it.

Reply:
@n01r is trying to get in contact with here.

Reply:
She agrees with the suggestion `Sophie Rudat (Koßagk)`